src/main/java/org/services/Cleaning.java

```diff
@@ -39,4 +53,25 @@ public String getDisplayName(){
     public Vehicle getVehicle(){
         return vehicle;
     }
+
+    @Override
+    public boolean remove(){
+        return this.getVehicle().activeServices.remove(this);
+    }
+
+    @Override
+    public boolean complete(){
```

Comment:
given the boolean in the signature this might be confusing  for the reader, are you:
a) compleating the service (as marking it done)
b) checking is the r service complete? return true/false